### PR TITLE
pyflakes cleanup

### LINKED ISSUE #2450: Remove wildcard imports from rpc-tests scripts to allow checking by pyflakes
``pyflakes`` can't be used on the Python scripts under the qa/rpc-tests directory because they use wildcard imports (``from ... import *``). Clean up the imports to make them checkable.

### PR DESCRIPTION
**Pull request by daira**
Make the RPC tests pyflakes-clean. fixes #2450 

Includes a fix that upstream had in https://github.com/bitcoin/bitcoin/pull/7802

### CODE REVIEW

**Comment by daira** [comment]
@zkbot try

**Comment by daira** [comment]
Rebased to include mempool_tx_input_limit.py which was added in #2342.

File: qa/rpc-tests/nodehandling.py [hunk 1]
```diff
@@ -8,13 +8,10 @@
 #
 
 from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import *
-import base64
+from test_framework.util import assert_equal, connect_nodes_bi, p2p_port
+
+import time
```
Comment (str4d) [inline qa/rpc-tests/nodehandling.py]: Huh, was `from test_framework.util import *` importing `time`?

Comment (daira) [inline qa/rpc-tests/nodehandling.py]: Yes, because util implicitly rexports it.

File: qa/rpc-tests/httpbasics.py [hunk 2]
```diff
@@ -36,66 +37,66 @@ def run_test(self):
...
+        assert_equal(conn.sock!=None, True) # according to http/1.1 connection must still be open!
 
-        #send 2nd request without closing connection
+        # send 2nd request without closing connection
         conn.request('POST', '/', '{"method": "getchaintips"}', headers)
-        out2 = conn.getresponse().read();
-        assert_equal('"error":null' in out1, True) #must also response with a correct json-rpc message
-        assert_equal(conn.sock!=None, True) #according to http/1.1 connection must still be open!
+        out2 = conn.getresponse().read()
+        assert_equal('"error":null' in out2, True) # must also response with a correct json-rpc message
```
Comment (daira) [inline qa/rpc-tests/httpbasics.py]: Added a reference to that in the PR description.

**Comment by daira** [comment]
Don't merge this until we've added the CI buildstep in #2494.

**Comment by bitcartel** [comment]
@daira Looks good.  What command line options did you run pyflakes with?

**Comment by str4d** [comment]
CI buildstep has been added.

@zkbot r+

**Pull request merged**